.github/workflows/fcos-podman-next-build.yml

```diff
@@ -0,0 +1,53 @@
+name: Build FCOS image with packages from rhcontainerbot/podman-next
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
After this PR merges, it's worth checking out the job on the 'actions' tab to see how it's triggering.  I *think* it only runs one for a 'merge' commit, and not once for every pushed commit.  You never can be sure with these GHA things, they really behave strangely sometimes :cry: